src/api/v1/public/routes/apps.js

```diff
@@ -55,16 +56,16 @@ app.get("/", function(req, res)
                 message: getSuccessMessageForGetApps(req.query),
                 data: data.results,
             });
-            res.send(response);
+            res.json(response);
```

Comment:
I want anyone receiving the response to know it's of type JSON. Forgot to do this before.